draft-demarco-oauth-status-attestations.md

```diff
@@ -175,10 +175,10 @@ The general requirements for the implementation of Status Assertion are
 listed in this section. The Status Assertion:
 
 - SHOULD be presented in conjunction with the Digital Credential.
-The Status Assertion MUST be timestamped with its issuance datetime,
+- MUST be timestamped with its issuance datetime,
 using a timestamp which is later then the time of presentation issuance;
 - MUST contain the expiration datetime after which the Digital Credential
-MUST NOT be considered valid anymore. The expiration datetime MUST be
+must not be considered valid anymore. The expiration datetime MUST be
```

Comment:
these was addressed in the PR https://github.com/peppelinux/draft-demarco-oauth-status-attestations/pull/58 abd must be ignored here, I'll take care about any conflict/alignment during the merge